src/main/java/org/terasology/climateConditions/ClimateConditionsSystem.java

```diff
@@ -56,28 +70,38 @@ public void addHumidityModifier(float order, ConditionModifier humidityModifier)
         humidityModifiers.put(order, humidityModifier);
     }
 
-    public void configureTemperature(int seaLevel, int maxLevel, float diversity, Function<Float, Float> function,
-                                     float minimumValue, float maximumValue) {
+    public void configureTemperature() {
+        configureTemperature(32, 10,-10, 10, world, biome);
+    }
+
+    public void configureHumidity() {
+        configureHumidity(32, 10, 0, 1, world, biome);
+    }
+
+    public void configureTemperature(int seaLevel, float diversity, float minimumValue, float maximumValue, WorldProvider world, BiomeRegistry biome) {
         int seed = worldSeed.hashCode();
-        
+
         float noiseMultiplier = minMultiplier + (maxMultiplier - minMultiplier) * diversity;
```

Comment:
I see that this is not part of the PR, but why are they called `minMultiplier` and `maxMultiplier` when diversity values being passed are greater than 1